commit cfa8e94e9574ef757d8f367cc00faa69d6468bf4
Author: Gregory Hainaut <gregory.hainaut@gmail.com>
Date:   Sun Jun 29 13:16:03 2014 +0200

    wx3.0: disable an useless symbol on linux
    
    Not used and print an error message for each plugins.

diff --git a/pcsx2/PluginManager.cpp b/pcsx2/PluginManager.cpp
index 6a6e719..19947d9 100644
--- a/pcsx2/PluginManager.cpp
+++ b/pcsx2/PluginManager.cpp
@@ -853,7 +853,11 @@ SysCorePlugins::PluginStatus_t::PluginStatus_t( PluginsEnum_t _pid, const wxStri
 
 	_PS2EgetLibName		GetLibName		= (_PS2EgetLibName)		Lib.GetSymbol( L"PS2EgetLibName" );
 	_PS2EgetLibVersion2	GetLibVersion2	= (_PS2EgetLibVersion2)	Lib.GetSymbol( L"PS2EgetLibVersion2" );
+#ifdef __LINUX__
+	_PS2EsetEmuVersion	SetEmuVersion	= NULL;
+#else
 	_PS2EsetEmuVersion	SetEmuVersion	= (_PS2EsetEmuVersion)	Lib.GetSymbol( L"PS2EsetEmuVersion" );
+#endif
 
 	if( GetLibName == NULL || GetLibVersion2 == NULL )
 		throw Exception::PluginLoadError( pid ).SetStreamName(Filename)
